modules/analytics/analytics.js

```diff
@@ -123,6 +123,13 @@ class Analytics {
                     permanentProperties.group = group;
                 }
 
+                if (window.jitsiRegionInfo) {
+                    permanentProperties.ProxyRegion;
```

Comment:
`permanentProperties.ProxyRegion;`
I don't get how is this supposed to work ?
Is it one of some magic expressions ?